phys/module_diag_trad_fields.F

```diff
@@ -112,6 +121,20 @@ SUBROUTINE trad_fields ( u,v,w,t,qv,zp,zb,pp,pb,p,pw,            &
                   END IF
                END IF
 
+               !  Potential Temperature  
+
+               IF ( use_theta_m .EQ. 1 ) THEN
+                  temperature(i,k,j) =  ( t(i,k,j) + t0 )  / ( 1. + (R_v/R_d) * qv(i,k,j) )
+               ELSE
+                  temperature(i,k,j) =   t(i,k,j) + t0  
+               END IF
+
+
+               !  Relative humidity
+
+                  ptot(i,k,j) = pb(i,k,j)+pp(i,k,j)
+                  rh(i,k,j) = calc_rh(ptot(i,k,j), temperature(i,k,j), qv(i,k,j)) 
```

Comment:
@smileMchen 
Ming,
Does your algorithm assume temperature or potential temperature?

Reply:
@davegill 
It is temperature.